src/icalendar/prop/__init__.py

```diff
@@ -1736,6 +2383,48 @@ def from_ical(ical):
         except Exception as e:
             raise ValueError(f"Expected time, got: {ical}") from e
 
+    @classmethod
+    def examples(cls) -> list[vTime]:
+        """Examples of vTime"""
+        return [cls(time(12, 30))]
+
+    from icalendar.param import VALUE
+
+    def to_jcal(self, name: str) -> list:
+        """The jcal represenation of this property according to :rfc:`7265`."""
+        value = self.dt.strftime("%H:%M:%S")
+        if self.params.is_utc():
+            value += "Z"
+        return [name, self.params.to_jcal(exclude_utc=True), self.VALUE.lower(), value]
+
+    @classmethod
+    def parse_jcal_value(cls, jcal: str) -> time:
+        """Parse a jcal string to a datetime.time.
+
+        Raises:
+            JCalParsingError
```

Comment:
```suggestion
        """Parse a jcal string to a :py:class:`datetime.time`.

        Raises:
            JCalParsingError: If it can't parse a time.
```